File: Codigos estudiantes por lenguaje/Java/Paula Rios/introducción/matrices.java
import java.util.*;
public class Main {
    public static void main(String[] args) {
        Scanner lector=new Scanner(System.in);
        long matrix[][]=new long[3][3];
        int cont=1;
        for (int i=0; i<3; i++) {
            for (int j=0; j<3; j++){
                matrix[i][j]=cont;
                cont++;
            }
        }
        for (int i=0; i<3; i++) {
            for (int j=0; j<3; j++){
                System.out.print(matrix[i][j]+"\t");
            }
            System.out.println("");
        }
            
    }   
}

File: Codigos estudiantes por lenguaje/C++/Paula Rios/Introduccion/matrices.cpp
#include <bits/stdc++.h>
#include <cstdlib>
using namespace std;
int main() {
	long matrix [8][7];
	int cont=1;
	for(int i=0;i<8;i++){
		for(int j=0;j<7;j++){
			matrix[i][j]=cont;
			cont++;
		}
	}
	for(int i=0;i<8;i++){
		for(int j=0;j<7;j++){
			cout<<matrix[i][j]<<"\t";
		}
		cout<<endl;
	}
}
